common/components/mlflow_model_local_validation/spec.yaml

```diff
@@ -20,6 +20,9 @@ command: |
   cp ${{inputs.model_path}}/conda.yaml ./
   conda env create -p $ENV_PREFIX -f conda.yaml -q
```

Comment:
This is likely not compatible with air-gapped regions (no access to conda channels). Please include any dependencies into the environment.

Reply:
@elesel, for this component, conda env has to created and validated. Could you please share how we can do that? Could we skip publishing import components to air-gap region if it needs to create an env beforehand?

Reply:
Also looks like this problem will be with evaluation component too, as they also create conda env on fly for the model and then do evaluation